addons/mail/static/src/models/notification_group/notification_group.js

```diff
@@ -101,6 +101,13 @@ function factory(dependencies) {
         id: attr({
             required: true,
         }),
+        /**
+         * Greatest `message.id` for this notification group. Used to sort by most
+         * recent message.
+         */
+        maxId: attr({
+            default: 0,
+        }),
```

Comment:
Greatest is a strange word to use in this context, biggest maybe?

But we can probably rename this into `sequence` instead, which is typically used as keyword for sorting keys.
The fact that it comes from ids is just a technical detail.